src/extensions/storage_macro/dbc.rs

```diff
@@ -0,0 +1,44 @@
+use async_recursion::async_recursion;
+
+use rdbc2;
+
+type Error = Box<dyn std::error::Error + Send + Sync + 'static>;
+
+impl crate::application::CoLink {
+    #[async_recursion]
+    async fn _search_and_generate_query_string(&self, address: &str, key_name: &str) -> Result<String, Error> {
```

Comment:
what is "address"? Shall we use "string_before_rdbc" which is more informative? Also key_name is actually string_after, right?

Reply:
Done.